pkg/odo/cli/component/update.go

```diff
@@ -57,6 +70,18 @@ func NewUpdateOptions() *UpdateOptions {
 
 // Complete completes update args
 func (uo *UpdateOptions) Complete(name string, cmd *cobra.Command, args []string) (err error) {
+	uo.devfilePath = filepath.Join(uo.componentContext, DevfilePath)
+
+	if experimental.IsExperimentalModeEnabled() && util.CheckPathExists(uo.devfilePath) {
+		// Add a disclaimer that we are in *experimental mode*
+		log.Experimental("Experimental mode is enabled, use at your own risk")
+
+		// Configure the context
```

Comment:
I think we are talking about configuring the devfile context. Can we change it to `Configure the devfile context` to be more specific

Reply:
this change was supposed to be a part of https://github.com/openshift/odo/pull/3453 . I included this here by mistake.